docs/design/batching.md

```diff
@@ -0,0 +1,64 @@
+# Batching VolumeSnapshotMover Backups and Restores
+
+## Summary
+The VolumeSnapshotMover controller currently runs on each `volumeSnapshotBackup` 
+and `volumeSnapshotRestore` until completed. The problem with this is that as 
+the number of VolSync processes increases, the time for backup/restore to complete
+can become slow, as diminishing returns increases.
+
+
+## Performance Testing Results
+[Performance testing](https://docs.google.com/document/d/1kPKo46_McEkj0Fu9hdkTxiVDnTh00forFQnscTgiEgg/edit?usp=sharing) 
+found that there is an optimal batch number of 12 for backup. For restore, 
+the results appear to be linear. 
+- It is important to note that these results are
+from a small set of factors such a CephFS and Ceph RBD, as well as small
+volume sizes. It is possible for this number to change with different scenarios.
+
+
+## Motivation
+Improve the performance of VolumeSnapshotMover backup and restore by allowing 
+users to specify a concurrent number of `volumeSnapshotBackups` and 
+`volumeSnapshotRestores` that can be simultaneously `inProgress` at once.
+
+
+## Implementation details
+Configurable *int64 values, `dpa.spec.dataMover.concurrentBackupVolumes` and 
+`dpa.spec.dataMover.concurrentRestoreVolumes`, can be used to specify 
```

Comment:
I'm not picky on names, but wondering if this should be `maxConcurrentVolumeBackups` or something to indicate it is the maximum number of allowable running backups